### PR TITLE
Formatters doc

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

**Comment by k-anderson** [comment]
@jamesaimonetti shouldn't we move these to a core library?  If they are in the current location they would be loaded in system_schemas, and I don't think there is any reason to do so...

**Comment by jamesaimonetti** [comment]
@k-anderson well, I debated that. Is there a reason not to load them, aside from storing them? Would someone like to access them via `/v2/schemas` for some reason? We could build a UI for creating formatters that would load `/v2/schemas/kapi.offnet_resource.req` to show you available keys for formatting, for instance.

Anyway, I didn't see a drawback to having them in the db / accessible via the API

File: applications/crossbar/priv/couchdb/schemas/kapi.dialplan.bridge.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+                "peer"
+            ],
+            "type": "string"
+        },
+        "Enable-T38-Passthrough": {
+            "type": "string"
+        },
+        "Endpoints": {
+            "items": {
+                "type": "string"
```
Comment (lazedo) [inline applications/crossbar/priv/couchdb/schemas/kapi.dialplan.bridge.json]: should this be `object` or `$ref` to `endpoint` ?

Comment (jamesaimonetti) [inline applications/crossbar/priv/couchdb/schemas/kapi.dialplan.bridge.json]: yes, i was going to go through and update some to $ref'd objects. Probably do that now :)

**Comment by jamesaimonetti** [comment]
@k-anderson appears something is up with erlydtl and building a release, not related to this PR.

**Comment by jamesaimonetti** [comment]
@k-anderson @lazedo still a merl / relx issue and unrelated to the PR; want to merge?

**Pull request merged**